.github/workflows/add-changelog-snippet.yml

```diff
@@ -0,0 +1,63 @@
+name: Add changelog.d snippet
+
+on:
+  pull_request_target:
+    # "labeled" is also triggered when a PR is created with one or more labels
+    types: [synchronize, labeled]
+
+# Prevent the workflow from running multiple jobs at once when a PR is created
+# with multiple labels:
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref_name }}
+  cancel-in-progress: true
+
+jobs:
+  add:
+    runs-on: ubuntu-latest
+    if: >
+      github.repository == 'datalad/datalad'
```

Comment:
since this is to prevent running this workflow on forks, and since we are unlikely to fork into datalad organization itself but might decide to adopt this workflow elsewhere,  please make it allow for any repository under `datalad` organization.

Reply:
Done: f5fe5aa54.